examples/IBLevelSet/ex0/example.cpp

```diff
@@ -614,8 +614,8 @@ main(int argc, char* argv[])
         const double rho_fluid = input_db->getDouble("RHO_F");
         const double rho_solid = input_db->getDouble("RHO_S");
         const double rho_gas = input_db->getDouble("RHO_G");
-        const int num_solid_interface_cells = input_db->getDouble("NUM_SOLID_INTERFACE_CELLS");
-        const int num_gas_interface_cells = input_db->getDouble("NUM_GAS_INTERFACE_CELLS");
+        const double num_solid_interface_cells = input_db->getDouble("NUM_SOLID_INTERFACE_CELLS");
+        const double num_gas_interface_cells = input_db->getDouble("NUM_GAS_INTERFACE_CELLS");
```

Comment:
Same question as before --- `num_` is most commonly used as an integer value in the library.